resources/js/Pages/Settings/Server.vue

```diff
@@ -1,94 +1,139 @@
 <script setup lang="ts">
-import { ref } from 'vue';
-import {Link} from "@inertiajs/vue3";
+import { defineProps, ref } from 'vue';
+import { Link, router, usePage } from '@inertiajs/vue3';
+import { useForm } from '@inertiajs/vue3';
+const { selected_server } = usePage().props;
+const serverId = selected_server?.id;
+const baseUrl = import.meta.env.VITE_APP_BASE_URL || '';
```

Comment:
Personally i would say this is better, because it just takes to window.origion which will never be null. Because this here right now, wont work, we dont have `VITE_APP_BASE_URL` environment variable.
```suggestion
const baseUrl = window.location.origin;
```